.github/workflows/database.yml

```diff
@@ -4,18 +4,18 @@ name: Validate Database
 on:
   pull_request:
     branches-ignore:
-      - 'l10n_master'
```

Comment:
Do we want to swap all. `'` for `"`?

Reply:
I am getting to that in https://github.com/bitwarden/server/pull/3684 and will just deal with the conflicts then. Didn't want to fight Prettier in this instance.

Reply:
It's my IDE with auto-format on save. Their linter is going after very different things and I think we should undo that exclusion ...